wwwapp/migrations/0059_migrate_away_from_python_allaccess.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-07 06:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+
+# After the migration is applied in production just comment out code depending on the allaccess library
+def forwards_func(apps, schema_editor):
+    AccountAccess = apps.get_model('allaccess', 'AccountAccess')
```

Comment:
This will fail initial migration after the app is removed.

RTFM https://docs.djangoproject.com/en/3.0/howto/writing-migrations/#migrating-data-between-third-party-apps

Reply:
Yeah... Removing dependencies in Django with migrations is a pain. We might just squash most of our migrations to get rid of this. Or maybe we will comment this out or something.

Reply:
I literally linked to a page that describes how are you supposed to do this properly. Basically, if it throws an "app not installed" exception we can safely assume there is no previous version to migrate from and skip this migration.